src/init.cpp

```diff
@@ -560,6 +561,10 @@ void SetupServerArgs(ArgsManager& argsman)
     hidden_args.emplace_back("-daemonwait");
 #endif
 
+#if defined(USE_SYSCALL_SANDBOX)
+    argsman.AddArg("-sandbox=<mode>", "Use the experimental syscall sandbox in the specified mode (\"disallow-and-log\", \"log-and-kill\" or \"kill\"). Allow only allowlisted (expected) syscalls to be used by bitcoind. Note that this is an experimental new feature that may cause bitcoind to exit or crash unexpectedly: use with caution. In the \"disallow-and-log\" mode an invokation of an unexpected syscall will be disallowed and logged. In the \"log-and-kill\" mode an invokation of an unexpected syscall will terminate bitcoind after logging the syscall violation. In the \"kill\" mode an invokation of an unexpected syscall will cause bitcoind to be killed immediately by the kernel.", ArgsManager::ALLOW_ANY, OptionsCategory::OPTIONS);
```

Comment:
Do the syscalls actually fail in the "disallow-and-log" mode? Would be good to describe it in the help a little more, or change it to a more suggestive name like "log-and-fail" if it does fail, or something like "log" or "log-warning" if it doesn't fail.

Also should s/invokation/invocation/

Reply:
Great feedback!

I've decided to remove the `disallow-and-log` mode: I think the modes `log-and-kill` and `kill` should be more than enough (to start with at least).

`disallow-and-log` was meant to disallow the invoked unexpected syscall (by not executing it), log syscall and otherwise continue execution. Note however that continued normal execution cannot be _guaranteed_: the exact behaviour depends on the specific error handling at the syscall invocation call site. In some cases a failed syscall will be handled gracefully and in some places it won't. The "may or may not continue execution" part makes this mode potentially very confusing in practice so I think it makes sense to simply remove it for now.

The typo has been fixed. Thanks!